src/locales/zh-HK.yaml

```diff
@@ -89,74 +93,99 @@ app:
       root_disabled: '{root} G代碼文件目錄不可用，請首先連接打印機或者檢查G代碼存放目錄的路徑與讀權限'
     url:
       klipper_config: 'https://www.klipper3d.org/zh-Hant/Config_Reference.html#%{hash}'
+      moonraker_config: 'https://moonraker.readthedocs.io/en/latest/configuration/#%{hash}'
+      moonraker_telegram_bot_config: 'https://github.com/nlef/moonraker-telegram-bot/wiki/Sample-config#%{hash}'
   gcode:
     btn:
       load_current_file: 載入當前文件
     label:
       current_layer_height: 當前層高
+      exclude_object: 排除對象
       follow_progress: 跟隨打印進度
       layer: 當前層數
       layers: 總層數
       parsed: 已解析
-      parsing_file:  解析文件
+      show_current_layer: 顯示當前層
       show_extrusions: 顯示已打印部分
       show_moves: 顯示移動走線
       show_next_layer: 顯示下一層
       show_previous_layer: 顯示上一層
       show_retractions: 顯示回抽
+      parsing_file: 解析文件
     msg:
       confirm: '文件 "%{filename}" 大小為 %{size}，這可能會導致系統短暫失去響應，是否確認？'
   general:
     btn:
+      abort: 中止
+      accept: 接受
       add: 添加
       add_dir: 新建目錄
       add_file: 新建文件
       add_printer: 添加打印機
+      add_to_queue: 添加到隊列
       adjust_layout: 調整儀表板布局
+      adjusted: 調整
       all: 全部
+      auth_unsure: 不確定為什麼會看到這個？
       calibrate: 校準
       cancel: 取消
       clear_profile: 清除配置文件
       close: 關閉
       config_reference: 配置參考
+      create_zip_archive: 壓縮
+      delete: 刪除
       download: 下載
       edit: 編輯
       exit_layout: 退出修改模式
       extrude: 擠出
+      filter: 過濾
+      forgot_password: 忘記密碼？
       heaters_off: 停止加熱
+      job_queue: ''
```

Comment:
There are 11 spots on this file with empty strings.

These need to either be translated or the key removed from the file (thus using the English fallback) so that we don't show empty labels!

![image](https://user-images.githubusercontent.com/85504/216564857-ce4475a5-4e1a-4e2c-bf20-ef4e8c288216.png)